web/docs/language/features.md

```diff
@@ -602,6 +602,193 @@ try {
+  entity: Task,
+  operations: {
+    getAll: {
+      isPublic: true, // optional, defaults to false
+    },
+    get: {},
+    create: {
+      overrideFn: import { createTask } from "@server/tasks.js", // optional
+    },
+    update: {},
+  },
+}
+```
+ 
+It has the following fields:
+- `entity: Entity` - the entity to which the CRUD operations will be applied.
+- `operations: { [operationName]: CrudOperationOptions }` - the operations to be generated. The key is the name of the operation, and the value is the operation configuration.
+  - The possible values for `operationName` are:
+    - `getAll`
+    - `get`
+    - `create`
+    - `update`
+    - `delete`
+  - `CrudOperationOptions` can have the following fields:
+    - `isPublic: bool` - Whether the operation is public or not. If it is public, it will be available to all users. If it is not public, it will be available only to authenticated users. Defaults to `false`.
+    - `overrideFn: ServerImport` - The import statement of the optional override implementation in Node.js.
+
+#### Defining the overrides
+
+Like with actions and queries, you define the implementation in a Javascript/Typescript file. The overrides are functions that take the following arguments:
+- `args` - The arguments of the operation i.e. the data that's sent from the client.
+- `context` - Context containing the `user` making the request and the `entities` object containing the entity that's being operated on.
+
+You can also import types for each of the functions you want to override from `@wasp/crud/{crud name}`. The types that are available are:
+- `GetAllQuery`
+- `GetQuery`
+- `CreateAction`
+- `UpdateAction`
+- `DeleteAction`
+
+If you have a CRUD named `Tasks`, you would import the types like this:
+```ts
+import type { GetAllQuery, GetQuery, CreateAction, UpdateAction, DeleteAction } from '@wasp/crud/Tasks'
+
+// Each of the types is a generic type, so you can use it like this:
+export const getAllOverride: GetAllQuery<Input, Output> = async (args, context) => {
+  // ...
+}
+```
+
+We'll show an example below.
+
+#### Using the CRUD operations in client code
+
+On the client, you import the CRUD operations from `@wasp/crud/{crud name}`. The names of the imports are the same as the names of the operations. For example, if you have a CRUD called `Tasks`, you would import the operations like this:
+
+```jsx title="SomePage.jsx"
+import { Tasks } from '@wasp/crud/Tasks'
+```
+
+You can then access the operations like this:
+```jsx title="SomePage.jsx"
+const { data } = Tasks.getAll.useQuery()
+const { data } = Tasks.get.useQuery({ id: 1 })
+const createAction = Tasks.create.useAction()
+const updateAction = Tasks.update.useAction()
+const deleteAction = Tasks.delete.useAction()
+
+// The CRUD operations are using the existing actions and queries
+// under the hood, so all the options are available as before.
+```
+
+We'll show usage in the example below.
+
+#### Full example
```

Comment:
Looking at all this, I wonder if guide is even needed? Seems like we have quite some duplication between here and guide.
I would consider one of two choices:
1. Nixing the guide.
2. Removing the duplication here and linking to guide for full example.

Reply:
I'll go for the second option since I believe that `Features` should be much shorter and link to detailed guides.